tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: default
+permalink: /tutorial/sars-cov2/
+---
+
+This tutorial shows you the basic of how to interact with V-pipe.
+
+For the purose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
+
+> **Organisation of Data** :
+>
+> V-pipe expects its samples data organized in a two-level hierarchy:
+>
+> - Input files to be grouped by samples (e.g.: patient samples or biological replicates of an experiment).
+> - A second level for distinction of datasets belonging to the same sample (e.g.: sample dates).
+> - inside, the directory `raw_data` hold the sequencing output in FASTQ format (optionally compressed with GZip)
+> - When in split files, paired-ends reads need to have `_R1` and `_R2` suffixes in their name.
+
+*[FASTQ]: [represents DNA sequencer reads along with quality scores](https://en.wikipedia.org/wiki/FASTQ_format)
+
+## Preparing a small dataset
+
+You can run the first test on your workstation or a good laptop.
```

Comment:
Is this line necessary?

Reply:
Niko even insisted that landing page make it obvious that smaller dataset can be run locally. (He is the one that suggested the "Hardware Requirement" approximation)
The point is to avoid scaring away potential users who don't have access to giant cluster.

(as opposed to say, some of the guys at the hackathon building _ginormous_ graph databases)